src/modules/roc_sndio/target_pulseaudio/roc_sndio/pulseaudio_backend.h

```diff
@@ -12,6 +12,8 @@
 #ifndef ROC_SNDIO_PULSEAUDIO_BACKEND_H_
 #define ROC_SNDIO_PULSEAUDIO_BACKEND_H_
 
+#include <pulse/pulseaudio.h>
```

Comment:
If pulseaudio headers are not used in `pulseaudio_backend.h`, it's better to move include to `pulseaudio_backend.cpp`, to reduce pollution of the global namespace of `pulseaudio_backend.h` users. This is a general rule for improving build time.